src/lib/webworker.generated.d.ts

```diff
@@ -5,14 +5,15 @@
 interface AddEventListenerOptions extends EventListenerOptions {
```

Comment:
Big changes, but this brings this file up-to-date with the `dom.d.ts` changes. WebWorker being opt-in meant we focused on the dom changes for the beta and these follow on.